extensions/variables-import-export/src/ui/Export.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import './app.css';
+
+  window.onmessage = ({data: {pluginMessage}}) => {
+    if (pluginMessage.type === 'EXPORT_RESULT') {
+      document.querySelector('textarea').innerHTML = pluginMessage.files
+        .map(
+          ({fileName, body}) =>
+            `/* ${fileName} */\n\n${JSON.stringify(body, null, 2)}`,
+        )
+        .join('\n\n\n');
+    }
+  };
+
+  function exportVariables() {
+    parent.postMessage({pluginMessage: {type: 'EXPORT'}}, '*');
```

Comment:
Suggestion: Wrap this `parent.postMessage` so you can add type checking for the arg to be a `CodeMessage`